.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: c
+sudo: required
+dist: trusty
+matrix:
```

Comment:
Hello Romanos,

I understand this proposal if the first step toward Travis CI
integration with zrpcd+quagga.
First step is for Ubuntu Trusty, but second step may be to Centos, for instance.
Right ?

Internally, we already have a CI that compiles and also does some testing.
But having travis will bring an added value for external people
wishing to use zrpcd.

So, I will accept the pull request.
Just in order to be compliant with other distros, I will check the
changed code against this point.